front-end/src/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.css';
+import DataEntry from './Components/DataEntry';
+// import Home from './Components/Home';
+import Login from './Components/Login';
+import Registration from './Components/Registration';
+import { Route, createBrowserRouter, createRoutesFromElements, RouterProvider, Outlet } from "react-router-dom";
+import Update from './Components/Update';
+import HomePage from './Components/HomePage';
+
+
+function App() {
+  const router = createBrowserRouter(createRoutesFromElements(
```

Comment:
This is correct but you should write JS Preferred code
```
<Router>
    <Routes>
        // Your Routes
        <Route />
     </Routes>
</Router>
```

Reply:
Okay I will look into it and change this